generate.py

```diff
@@ -395,6 +395,13 @@ def decode_n_tokens(
                     )
                     input_pos += 1
                     break
+            if _i == 1:
```

Comment:
Remove this before final landing.  It was to do more accurate tokens/sec measurement during development, especially for torch.compile.